travis-scripts/valgrind.sh

```diff
@@ -165,8 +165,9 @@ echo "Checking that serverd and execd PIDs are still correct/alive:"
 ps -p $exec_pid
 ps -p $server_pid
 
-echo "Killing valgrind"
+echo "Killing valgrind cf-execd"
 kill $exec_pid
+echo "Killing valgrind cf-serverd"
```

Comment:
Also unrelated.

Reply:
Changed commits to better reflect this change.